File: backend/src/main/java/edu/cit/fitflow/dto/UserMealDTO.java
package edu.cit.fitflow.dto;

import lombok.Data;

@Data
public class UserMealDTO {
    private Integer userId;
    private String name;
    private String time;
    private Integer calories;
    private Integer protein;
    private Integer carbs;
    private Integer fats;
    private String description;
    private String notes;
    private String ingredients;
    private String image;
}

File: frontend_mobile/app/src/main/java/edu/cit/fitflow/UserMealDTO.kt
package edu.cit.fitflow

data class UserMealDTO(
    val userId: Int,
    val name: String,
    val time: String,
    val calories: Int,
    val protein: Int,
    val carbs: Int,
    val fats: Int,
    val notes: String,
    val ingredients: String,
    val image: String // You can use a URL or base64-encoded string depending on how you handle images
)
